### PR TITLE
Sec fixes

### PR DESCRIPTION
**Pull request by cohenaj194**
ref: https://github.com/ff14-advanced-market-search/temp-fe/issues/6

### Summary by CodeRabbit

**Release Notes for Pull Request**

- **Style**: Enhanced user interface by highlighting example values in form fields across multiple templates (`bestdeals.html`, `ffxiv_craftsim.html`, `ffxiv_craftsimconfig.html`, `ffxiv_shoppinglist.html`, `ffxivbestdeals.html`, `petexport.html`, `petimport.html`, `petmarketshare.html`, `petshoppinglist.html`, `regionundercut.html`). This change improves clarity and readability for users interacting with the forms.
- **Chore**: Updated various dependencies in `requirements.txt` to their latest versions, ensuring the application stays up-to-date with the most recent improvements and security patches.
- **Refactor**: Improved security in `app.py` by adding several security headers to the response in the `root()` function.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -28,7 +28,49 @@ def str_to_bool(bool_str):
...
+#         "connect-src": ["'self'"],
+#         "frame-src": ["'self'", "https://www.youtube.com"],
+#     }
+#     csp_header_value = "; ".join(
+#         [f"{key} {' '.join(value)}" for key, value in csp_policy.items()]
+#     )
+#     response.headers["Content-Security-Policy"] = csp_header_value
+#     return response
+
+#### WIP ####
```
[the remark this answers is not in the document]
Comment (cohenaj194) [reply app.py:73]: Yes this breaks the ajax charts I use at the end of the code.  How can i fix it?

File: templates/bestdeals.html
Comment (luan-dev) [inline templates/bestdeals.html:1]: personally i would recommend using `code` just for readability, but this is a preference thing

File: templates/ffxiv_craftsim.html
Comment (luan-dev) [inline templates/ffxiv_craftsim.html:1]: same here. having the styling and color isn't human-readable

File: templates/ffxiv_craftsimconfig.html
Comment (luan-dev) [inline templates/ffxiv_craftsimconfig.html:1]: same

**Review by luan-dev — CHANGES_REQUESTED** [review]

**Pull request merged**